vagrant/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -4,7 +4,29 @@
     name: samba
     state: present
 
+- name: Prepare CephFS VFS support
+  when: config.be.variant == 'vfs'
+  block:
+    - name: Install CephFS VFS module
+      yum:
+        name: samba-vfs-cephfs
+        state: present
+
+    - name: Set selinux in permissive mode
+      selinux:
+        policy: targeted
+        state: permissive
```

Comment:
Moving forward we will find ways(policies, allow rules, booleans etc) to make Samba-CephFS integration via VFS module work in default _Enforcing_ mode.